docs/getting_started/online_comp_getting_started.md

```diff
@@ -0,0 +1,84 @@
+#Getting Started on the Online Compiler
+
+The mbed Online Compiler delivers a full online editor, standard keyboard shortcuts, flexible workspace management and version control optimized for the mbed ecosystem.
+
+This article reviews the basics of interacting and creating and maintaining code in your workspace.
+
+##Interface basics
+The mbed Online Compiler interface consists of five key components - [main toolbar](#main-toolbar), [navigation tree](#navigation-tree), [browser panel](#browser-panel), [editor](/#editor) and [output panel](#output-panel).
+
+###Main toolbar
+The buttons on the main toolbar are accessible at all times though they may not always be active because their state depends on the current context or current task. For example, when you're browsing a program, a folder or a library, the "Undo", "Redo", "Print" and "Format Code" buttons won't be active because their function is relative to text editing. Therefore, you must select a file in the navigation tree for them to be active. Similarly, selecting "My Programs" in the navigation tree disables "Compile", "Commit", "Revision History" and other buttons. "My Programs" is your workspace root. Thus, you can't complete these tasks on it. If unsure of the context you are currently working on, you can always look at the top blue bar that shows your location.
+
+Most of the main toolbar buttons have a keyboard [shortcut](/online_comp.md#keyboard-shortcuts), which saves a mouse click or two. Some buttons have a dropdown menu with similar or related functions. For example, there are several ways to compile under "Compile" and different new items under "New".
+
+When you edit a file, the main toolbar looks like the image below. (Click to enlarge.)
+
+![](/images/main-toolbar.png)
+
+On the right side of the main toolbar is the device selection dropdown, which lets you change the device you are compiling for. The mbed Compiler Guide is accessible through the top-right Guide button or via F1 (Ctrl+G alternatively) keyboard shortcut.
+
```

Comment:
Note: I could not get "Ctrl+G" to work. Please confirm.
Second note: I did not see a "Guide" button in the top right, but I did see a "Help" button. Please confirm.